### ISSUE TITLE
error in Sphinx build

### ISSUE THREAD

**Issue by prjemian** [NSLS-II/ophyd#615]
During work on another issue (#614), a bug in the documentation on the master branch was discovered:

```
Warning, treated as error:
/installed_path/BlueSky/lib/python3.6/site-packages/ophyd/areadetector/base.py:docstring of ophyd.areadetector.base.ADBase:1:Literal block expected; none found.
```

**Comment by prjemian** [comment]
This issue will be addressed as part of the #614 branch effort.

**Comment by prjemian** [comment]
Seems there is more to test building the ophyd documentation than these steps:

```
cd ophyd/doc
make clean
make html
```

and this additional complexity is the cause of the reported error.  Abandoning this issue for now.
